pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -412,6 +408,8 @@ func (p *Provisioner) setObjectContext() error {
 		return errors.Wrapf(err, "failed to set multisite on provisioner's objectContext")
 	}
 
+	p.objectContext.Endpoint = store.Status.Info["endpoint"]
+
```

Comment:
@leseb I have one more doubt, do we need to objectContext.Endpoint like in ceph object user?

Reply:
Yes, Noobaa needs it, but we should also do it if `p.objectContext.Endpoint` is not empty, can you do that change?

Reply:
Sure will update the patch